libs/datasets/sources/fips_population.py

```diff
@@ -76,60 +76,12 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         states_aggregated[cls.Fields.FIPS] = states_aggregated[cls.Fields.STATE].map(ABBREV_US_FIPS)
         states_aggregated[cls.Fields.COUNTY] = None
 
-        common_fields_data = pd.concat([data, states_aggregated])
-        return common_fields_data
-
+        us_row = {
+            CommonFields.FIPS: "0",
+            CommonFields.COUNTRY: "USA",
+            CommonFields.POPULATION: states_aggregated[CommonFields.POPULATION].sum(),
```

Comment:
I think this makes sense - should we double check that our number matches other public US population numbers?

Reply:
I took a look at https://www.census.gov/popclock/ which says it is 50 states and DC. Looking at 
```
geo_df = pd.read_csv('data/geo-data.csv')
static_df = pd.read_csv('data/multiregion-static.csv').merge(geo_df, on='location_id')
static_df.loc[(static_df['aggregate_level'] == 'state') & static_df['state'].isin(set(us_state_abbrev.STATES_AND_DC.values()))]['population'].sum()
```
we say 328.2M. Picking the date on https://www.census.gov/popclock/ I need to go back to about June 1 2019 to get that.
OTOH this PR doesn't modify the population. `multiregion-static` isn't changed.
@mikelehen Please make a new trello card if you think it is worth looking into this further.